typing/typedecl.ml

```diff
@@ -1869,7 +1900,8 @@ let transl_with_constraint id ?fixed_row_path ~sig_env ~sig_decl ~outer_env
      declaration [sdecl] in the outer environment [outer_env]. *)
   let env = outer_env in
   let loc = sdecl.ptype_loc in
-  let tparams = make_params env sdecl.ptype_params in
+  (* CR aspectorzabusky: Log that it's from a constraint? *)
```

Comment:
I think so -- otherwise the error message is going to be very confusing.

But actually I'm not sure that an error can occur here. I don't think we allow e.g. `S with type ('a : immediate) t = ...`. Would any layout checks on the RHS of the `=` be ascribed this reason?